src/lib.rs

```diff
@@ -194,6 +195,11 @@ pub fn all(args: TokenStream, input: TokenStream) -> TokenStream {
     cfg("all", args, input)
 }
 
+#[proc_macro_attribute]
+pub fn minver(args: TokenStream, input: TokenStream) -> TokenStream {
+    cfg("minver", args, input)
```

Comment:
Ideally, this would be only usable as a global `#![attr]` at the crate level. Is it possible to restrict that?